2022 - Horus Heresy.gst

```diff
@@ -123,9 +123,14 @@
             <modifier type="set" field="name" value="It Will Not Die (5+)"/>
           </modifiers>
         </infoLink>
-        <infoLink id="2a4b-7961-1ac1-06d9" name="Bulky (X) (P3P)" hidden="false" targetId="676c-7b75-4b6f-9405" type="rule">
+        <infoLink id="2a4b-7961-1ac1-06d9" name="Bulky (X)" hidden="false" targetId="676c-7b75-4b6f-9405" type="rule">
           <modifiers>
             <modifier type="set" field="name" value="Bulky (4)"/>
+            <modifier type="set" field="hidden" value="true">
```

Comment:
Added condition to hide this when using EC so Fulgrim can show Bulky (6) by itself

Reply:
I believe we tend to use "set name to bulky (6)" for this type of mod.

Reply:
That is on Fulgrim's profile. This is to hide the original Bulky (4) which is printed off for all Primarchs. Otherwise you get a Bulky (4) and Bulky (6) on Fulgrim's rules.